blending/raymond/balance.f90

```diff
@@ -0,0 +1,148 @@
+      subroutine balance(ml,nl,kk1, &
```

Comment:
Is this balance.f90 file actually used?

Reply:
No, I am currently not using it. I will remove it.

Some context: That was part of the code that I received, but it is unclear if it is needed and I haven't been using it. Probably best to leave it out.

Reply:
Thanks for the change!